kolibri/core/assets/src/views/core-base.vue

```diff
@@ -103,6 +103,7 @@
         return {
           top: `${this.headerHeight}px`,
           [this.isRtl ? 'right' : 'left']: 0,
+          padding: (this.mobile ? 16 : 32) + 'px',
```

Comment:
Would need to update this to:
```
paddingLeft: (this.mobile ? 16 : 32) + 'px',
paddingRight: (this.mobile ? 16 : 32) + 'px',
```
Otherwise the bottom padding in the CSS block below will get overwritten by the blanket `padding` style.

Reply:
My bad. Now it has been updated and pushed.